### DISCUSSION TITLE
Can't use `is:archived` in org PR search

### DISCUSSION THREAD

**Discussion by l0b0** [community/community#71901] [Pull Requests]
### Select Topic Area

Bug

### Body

#8591 has been answered, but doesn't work everywhere. When attempting to [search for archived repos in my org](https://github.com/pulls?q=is%3Apr+author%3Aapp%2Fdependabot+org%3Alinz+is%3Aopen+is%3Aarchived), the `is:archived` part of the search is excluded.

**Comment by look** [comment]
I'm going to move this to the [Pull Requests](https://github.com/orgs/community/discussions/categories/pull-requests) category. Unfortunately, not all content types support `is:archived`. https://github.com/orgs/community/discussions/8591 is about code search specifically.
